<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientSearchForm</class>
 <widget class="QWidget" name="PatientSearchForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>101</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поиск больных</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Поле поиска:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_type"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_request"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_ok">
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
